### PR TITLE
change form design

### PR DESCRIPTION
**Pull request by Zakiamangal**

### CODE REVIEW

**Review by sahdasamier — CHANGES_REQUESTED** [review]
dont change or add any field , you play with UI only , so please change in text inside the field and the size of the field , like we discuse in the meeting

**Review by sahdasamier — CHANGES_REQUESTED** [review]
Zakia, there are more lines here that mean it is wrong it should be few lines that that

File: src/components/Form/index.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ export const Form = ({
         toggleSnackbar(true)
         setSnackbarMessage(message)
         if (isSuccess) {
-          navigate('/feed')
+          navigate('feed')
```
Comment (sahdasamier) [inline src/components/Form/index.tsx:134]: dont change in navigation

File: src/components/Form/index.tsx [hunk 2]
```diff
@@ -188,245 +188,302 @@ export const Form = ({
...
+      'Signed claim',
+      'Blockchain',
+      'Physical document',
+      'Integration'
     ]
   }
 
   return (
     <>
-      <DialogTitle>
```
Comment (sahdasamier) [inline src/components/Form/index.tsx:205]: you made changes in the function it self , please change only in the design and text

File: src/components/Form/index.tsx (hunk 1 above)
Comment (Zakiamangal) [reply src/components/Form/index.tsx:134]: ita bez of conflict i will correct it

File: src/components/Form/index.tsx (hunk 2 above)
Comment (Zakiamangal) [reply src/components/Form/index.tsx:205]: when i was to remove conflict so this happens i will correct it

**Review by sahdasamier — CHANGES_REQUESTED** [review]

File: src/components/Form/index.tsx [hunk 3]
```diff
@@ -188,245 +188,275 @@ export const Form = ({
...
-          }}
-        >
-          Submit
-        </Button>
-        {!!onCancel && <Button onClick={onCancel}>Cancel</Button>}
-      </DialogActions>
+                  )}
+                </>
+              )}
+              <LocalizationProvider dateAdapter={AdapterDateFns}>
```
Comment (sahdasamier) [inline src/components/Form/index.tsx:426]: there are an issue in the date , it is not working , please find the problem , it should be here

**Review by sahdasamier — CHANGES_REQUESTED** [review]
please make it responsive too, it is to good in the second form in the graph

**Pull request closed without merging**